### ISSUE TITLE
Postpone some initialization steps

### ISSUE THREAD

**Issue by marco-c** [mozilla/j2me.js#1168]
For example, we should be able to start the JVM before initializing the FS.
We could start initializing the FS at startup, start the JVM in parallel and, at some point, wait for the FS initialization to be finished before continuing.
This would allow us to do useful work while waiting for I/O.

**Comment by marco-c** [comment]
https://github.com/mozilla/j2me.js/pull/1178 - Loads emoji when a MIDlet goes in the foreground instead of right at startup
https://github.com/mozilla/j2me.js/pull/1201 - Makes the VM start before the MIDlet JAR and the filesystem are loaded
